bin/sync_ap_tier1_projects.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+#
+# Copyright 2021-2021 Ghent University
+#
+# This file is part of vsc-administration,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/hpcugent/vsc-administration
+#
+# All rights reserved.
+#
+"""
+This script synchronises the users and projects from the Resource App to the Ac
+
+FIXME: For now, it takes its data fom an ini file.
+
+The script must result in an idempotent execution, to ensure nothing breaks.
+"""
+from collections import namedtuple
+
+import logging
+import sys
+
+from configparser import ConfigParser
+
+from vsc.accountpage.sync import Sync
+from vsc.config.base import GENT, VSC_SLURM_CLUSTERS, PRODUCTION, PILOT
+
+
+VSC_ADMIN_GROUPS = ("gadminforever", "badmin", "l_sysadmin", "gt1_dodrio_vscadmins")
+
+AP_ACTIVE_USERS_AUTOGROUP = "gt1_dodrio_activeusers"
+
+
+ProjectIniConfig = namedtuple("ProjectIniConfig",
+    ["name", "description", "end_date", "members", "moderators"]
+)
+
+def get_projects(projects_ini):
+    """
+    This reads the ini file which contains information on the projects
+
+    [project_name]
```

Comment:
where do you define the core/gpu hours of a project?

Reply:
Right now in the pan file that holds the source. soonish in the web app for the resource tracking.